packages/core/src/App/Containers/Onboarding/cfds-platforms.jsx

```diff
@@ -0,0 +1,234 @@
+import React from 'react';
+import { withRouter } from 'react-router';
+import { Button, Text, Table, Icon, DesktopWrapper, MobileWrapper } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { routes } from '@deriv/shared';
+
+const data = [
+    {
+        title: 'Operating Systems',
+        dmt5: [
+            { title: 'iOS', icon: 'IcIos' },
+            { title: 'Android', icon: 'IcAndroid' },
+            { title: 'Windows', icon: 'IcWindowsGray' },
+            { title: 'macOS', icon: 'IcMacos2' },
+            { title: 'Linux', icon: 'IcLinux2' },
+            { title: 'Browser', icon: 'IcBrowser' },
+        ],
+        derivx: [
+            { title: 'iOS', icon: 'IcIos' },
+            { title: 'Android', icon: 'IcAndroid' },
+            { title: 'Browser', icon: 'IcBrowser' },
+        ],
+        has_items: true,
+    },
+    {
+        title: 'Asset classes',
+        dmt5: [
+            {
+                title: 'Synthetics',
+                icon: 'IcMt5Synthetics',
+            },
+            {
+                title: 'Cryptos',
+                icon: 'IcMt5Cryptos',
+            },
+            {
+                title: 'Indices',
+                icon: 'IcMt5Cryptos',
+            },
+            {
+                title: 'Stocks',
+                icon: 'IcMt5Stocks',
+            },
+            {
+                title: 'Forex',
+                icon: 'IcMt5Forex',
+            },
+            {
+                title: 'Commodities',
+                icon: 'IcMt5Commodities',
+            },
+        ],
+        derivx: [
+            {
+                title: 'Synthetics',
+                icon: 'IcMt5Synthetics',
+            },
+            {
+                title: 'Cryptos',
+                icon: 'IcMt5Cryptos',
+            },
+            {
+                title: 'Indices',
+                icon: 'IcMt5Cryptos',
+            },
+            {
+                title: 'Stocks',
+                icon: 'IcMt5Stocks',
+            },
+            {
+                title: 'Forex',
+                icon: 'IcMt5Forex',
+            },
+            {
+                title: 'Commodities',
+                icon: 'IcMt5Commodities',
+            },
+        ],
+        has_items: true,
+    },
+    {
+        title: 'Assets',
+        dmt5: '190+',
+        derivx: '190+',
+        has_items: false,
+    },
+    {
+        title: 'Leverage',
+        dmt5: 'Up to 1:1000',
```

Comment:
shouldn't we localize this?

Reply:
Yes, they are passed to localize inside the render function

Reply:
AFAIK we must pass strings directly to localize function. at least in binary we can't pass variables to localize